src/main/java/com/microsoft/aad/msal4j/MsalThrottlingException.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.aad.msal4j;
+
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+/**
+ * Exception type thrown when service returns throttling instruction:
+ * Retry-After header, 429 or 5xx statuses.
+ */
+@Accessors(fluent = true)
+@Getter
+public class MsalThrottlingException extends MsalServiceException {
```

Comment:
As stated above, ideally we wouldn't expose a new type of exception, but just rethrow an old MsalServiceExcpetion.

Reply:
bad idea